app/src/main/java/com/example/memeapp/data/local/LocalMemeDataBaseImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.memeapp.data.local
+
+import com.example.memeapp.data.local.model.MemeRealmEntity
+import com.example.memeapp.data.mapper.MemeRealmMapper
+import com.example.memeapp.data.mapper.MemeRealmMapper.transformRealmListOfMemes
+import com.example.memeapp.domain.entity.MemesEntity
+import com.example.memeapp.domain.local.LocalMemeDataBase
+import io.realm.Realm
+
+class LocalMemeDataBaseImpl(private val mapper: MemeRealmMapper) : LocalMemeDataBase {
```

Comment:
this variable mapper is not used here. Please remove it

Reply:
Done thx